### DISCUSSION TITLE
Is there any post-processing code for c++?

### DISCUSSION THREAD

**Discussion by xingjunhong** [open-mmlab/mmdetection#9042] [Q&A]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 🐞 Describe the bug

Is there any post-processing code for c++?

### Environment

Is there any post-processing code for c++?

**Comment by ZwwWayne** [comment]
MMDet is mainly based on PyTorch, thus written in Python. You may refer to MMDeploy, which exports the models in MMDet for real-world applications, which implements SDK for model inference, which should implement C++ code for the post-processing.

**Reply by xingjunhong** [reply]
Can you provide the c++ code location for the specific implementation? I can't find
